### PR TITLE
PR for Questions Page and Result page

### PR DESCRIPTION
**Pull request by manojasarsa**
It includes two parts:
- MCQ based Quiz Question Page
- Result Page with Answers & Score

### CODE REVIEW

File: ques-result/ques/ques.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+.ques_btn_box .quit_btn:hover {
+    color: var(--red-color);
+    transition: all 0.1s ease-in-out;
+}
+
+.ques_btn_box .next_btn:hover {
+    color: var(--blue-color);
+    transition: all 0.1s ease-in-out;
+}
```
Comment (BelideNikhil) [inline ques-result/ques/ques.css:79]: save the transition as a variable

**Review by BelideNikhil — APPROVED** [review]
good to go!

File: ques-result/ques/ques.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+.question {
+    padding: 1rem 0;
+    font-weight: 600;
+}
+
+.options {
+    justify-content: center;
+    align-items: center;
+}
```
Comment (adidoshi) [inline ques-result/ques/ques.css]: We can remove these lines of code, as it does not change anything.

**Review by adidoshi — APPROVED** [review]
Rest looks good!

**Pull request merged**